code/modules/mob/living/carbon/xenomorph/abilities/xeno_action.dm

```diff
@@ -160,14 +160,14 @@
 			return // We clicked the same ability in a very short time
 		if(xeno.client && xeno.client.prefs && xeno.client.prefs.toggle_prefs & TOGGLE_ABILITY_DEACTIVATION_OFF)
 			return
-		to_chat(xeno, "You will no longer use [ability_name] with \
+		to_chat(xeno, "We will no longer use [ability_name] with \
```

Comment:
Uh this is more of an OOC message about preferences I don't think this needs changing as it's addressing the player.